runHarness/Parameters.pm

```diff
@@ -2420,119 +2429,109 @@ $parameters{"msgServerSuffix"} = {
+"This is the directory that contains the jar file for the workload-driver executable relative to the current working directory.",
 	"showUsage" => 0,
 };
 $parameters{"workloadProfileDir"} = {
 	"type"    => "=s",
 	"default" => "workloadConfiguration",
 	"parent"  => "workloadDriver",
 	"usageText" =>
-"This is the directory that contains the templates for the workload profiles.\n\tIt must either include the full path to the directory\n\tor be relative to weathervaneHome.",
+"This is the directory that contains the templates for the workload profiles relative to the current working directory.",
 	"showUsage" => 0,
 };
 $parameters{"gcviewerDir"} = {
 	"type"    => "=s",
 	"default" => "",
 	"parent"  => "runManager",
 	"usageText" =>
-"This is the path to the gcViewer executable.  GcViewer is used to\n\tanalyze the Java Garbage-Collection logs.\n\tIt must either include the full path to the directory\n\tor be relative to weathervaneHome.",
+"This is the path to the gcViewer executable relative to the current working directory.\n\tGcViewer is used to analyze the Java Garbage-Collection logs.",
 	"showUsage" => 0,
 };
 $parameters{"resultsFileDir"} = {
 	"type"    => "=s",
 	"default" => "",
 	"parent"  => "runManager",
 	"usageText" =>
-"This is the directory in which Weathervane stores the csv summary file.\n\tIt must either include the full path to the directory\n\tor be relative to weathervaneHome.",
+"This is the directory in which Weathervane stores the csv summary file relative to the current working directory.",
 	"showUsage" => 0,
 };
 $parameters{"resultsFileName"} = {
 	"type"    => "=s",
 	"default" => "weathervaneResults.csv",
 	"parent"  => "runManager",
 	"usageText" =>
-"This is the name of the file in which Weathervane stores a summary of the run results in csv format.\n\tIt must either include the full path to the directory\n\tor be relative to weathervaneHome.",
+"This is the name of the file in which Weathervane stores a summary of the run results in csv format.",
 	"showUsage" => 1,
 };
 $parameters{"configDir"} = {
 	"type"    => "=s",
 	"default" => "configFiles",
 	"parent"  => "runManager",
 	"usageText" =>
-"This is the directory under which Weathervane stores\n\t configuration files for the various services.\n\tIt must either include the full path to the directory\n\tor be relative to weathervaneHome.",
+"This is the directory under which Weathervane stores configuration files for the various services relative to the current working directory.",
 	"showUsage" => 0,
 };
 $parameters{"dbLoaderImageDir"} = {
 	"type"    => "=s",
 	"default" => "images",
 	"parent"  => "dataManager",
 	"usageText" =>
-"This is the directory in which the images used by the dbLoader are stored.\n\tIt must either include the full path to the directory\n\tor be relative to weathervaneHome.",
+"This is the directory in which the images used by the dbLoader are stored relative to the current working directory.",
```

Comment:
This parameter can actually be removed since the images are now stored in the data manager's docker image